doc/cephfs/index.rst

```diff
@@ -33,8 +33,19 @@ and Swift APIs, or native bindings (librados).
 Using CephFS
 ============
 
-Using the Ceph Filesystem requires at least one :term:`Ceph Metadata Server` in
-your Ceph Storage Cluster.
+Prerequisites for using CephFS with a running Ceph Storage Cluster:
+
+- At least one active :term:`Ceph Metadata Server`.
+- Minimum one data and metadata :term:`RADOS` pool.
+
+.. note::
+
+         If cephx authentication is enabled, secret key is required for
+         mounting CephFS. It is a file containing key extracted from keyring.
```

Comment:
Only the kernel client needs the key extracted, and that won't even be required once this is merged: https://github.com/ceph/ceph/pull/29817